s3tests_boto3/functional/test_s3.py

```diff
@@ -905,7 +986,10 @@ def test_bucket_listv2_prefix_basic():
     bucket_name = _create_objects(keys=key_names)
     client = get_client()
 
-    response = client.list_objects_v2(Bucket=bucket_name, Prefix="foo/")
+    req_params = dict(Bucket=bucket_name, Prefix="foo/")
+    with allure.step(f"Request list_objects: {req_params}"):
```

Comment:
it is better to wrap client.list_objects_v2 and similar methods with an allure specific code, cause too many duplicates here

Reply:
perhaps it is even worth to just create a wrapper for a client class with getattribute overridden, and intercept there every call to methods and log input params and output from them to allure

Reply:
Opened a new PR #57 with client class methods overridden. 
Closing this one.